packages/authentication/types/allAuth.ts

```
@@ -0,0 +1,492 @@
+// import { MfaMethod } from './mfa'
+import { AxiosResponse } from 'axios'
+
+// Settings
+export enum Client {
+  APP = 'app',
+  BROWSER = 'browser',
+}
+
+// Config Types
+
+export interface ConfigRequest {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)